scripts/utils.sh

```diff
@@ -115,9 +117,45 @@ function setNginxHostnames {
     export ODOO_HOSTNAME="localhost:8069"
     export SENAITE_HOSTNAME="localhost:8081"
     export SUPERSET_HOSTNAME="localhost:8088"
+    export ERP_NEXT_HOSTNAME="localhost:8082"
     echo "→ O3_HOSTNAME=$O3_HOSTNAME"
     echo "→ ODOO_HOSTNAME=$ODOO_HOSTNAME"
     echo "→ SENAITE_HOSTNAME=$SENAITE_HOSTNAME"
     echo "→ SUPERSET_HOSTNAME=$SUPERSET_HOSTNAME"
+    echo "→ ERP_NEXT_HOSTNAME=$ERP_NEXT_HOSTNAME"
 
 }
+
+function displayAccesURLsWithCredentials() {
+    services=()
+    is_defined=()
+    
+    while read -r line; do
```

Comment:
A quick comment line to explain the logic will be best.
```
# read docker-compose-files.txt and extract the list of services run
```
or smtg of that sort.